developer/src/kmc-ldml/test/disp.tests.ts

```diff
@@ -67,25 +67,25 @@ describe('disp', function () {
     const disp = await loadSectionFixture(DispCompiler, 'sections/disp/invalid-dupto.xml', compilerTestCallbacks) as Disp;
     assert.isNull(disp);
     assert.equal(compilerTestCallbacks.messages.length, 1);
-    assert.deepEqual(compilerTestCallbacks.messages[0], LdmlCompilerMessages.Error_DisplayIsRepeated({ output: 'e' }));
+    assert.deepEqual(compilerTestCallbacks.messages[0], LdmlCompilerMessages.Error_DisplayIsRepeated({ output: 'e' }, withOffset(330)));
```

Comment:
using the "modern" test helpers we don't need to include the offset here (as the offset is removed unless requested), but it's not bad to have a few tests that actually check the offset, though it makes them more fragile if that .xml changes.